### DISCUSSION TITLE
I downloaded jbrowse2 using the command line interface, but encountered an error that prevented the page from loading.

### DISCUSSION THREAD

**Discussion by Opticalproperty** [GMOD/jbrowse-components#3684] [Q&A] [answered]
Hey there, jbrowse2 developers!

I recently downloaded jbrowse2 using the command line interface and attempted to launch it with the npx serve command. Unfortunately, I encountered an error message that popped up on the page. Can you help me troubleshoot this issue?

![image](https://user-images.githubusercontent.com/116154971/236419105-f601da60-fa51-486a-b08d-b9b73bd9f63f.png)

**Comment by cmdcolin** [comment]
@Opticalproperty we saw an issue here that was sort of similar where maybe our CLI tool was not unzipping the release correctly: https://github.com/GMOD/jbrowse-components/discussions/3657#discussioncomment-5688061


you can manually download from here https://github.com/GMOD/jbrowse-components/releases/download/v2.5.0/jbrowse-web-v2.5.0.zip and unzip it locally, that is all the jbrowse create/upgrade commands do, but we may want to investigate this issue to see if we can fix jbrowse create/upgrade

can you paste the output of `jbrowse --version`

**Reply by Opticalproperty** [reply]
![image](https://user-images.githubusercontent.com/116154971/236592833-0eaced24-9753-4276-863b-9bd725c47cab.png)
This is the version.
and I can use it in this zip file. Thanks!

**Reply by nathanweeks** [reply]
See https://github.com/GMOD/jbrowse-components/discussions/3657

**Comment by adrbmdns** [comment]
i have the same issue and download the zip file directly resolved the issue.

**Reply by cmdcolin** [reply]
thanks for confirming...will get the fix out in next release hopefully soon!

**Comment by cmdcolin** [comment, marked as answer]
v2.6.1 released with hopefully a fix for this issue! re-install the CLI tool with `npm install -g @jbrowse/cli` to get the update

**Reply by Opticalproperty** [reply]
thanks!
